### PR TITLE
ci/new issue templates

### PR DESCRIPTION
**Pull request by JeremyDec**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-8516
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] ~~Standalone app was ran and tested locally~~
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

You can check the rendering of the templates here:
- https://github.com/ovh/manager/blob/ci/new-issue-templates/.github/ISSUE_TEMPLATE/bug_report.yml
- https://github.com/ovh/manager/blob/ci/new-issue-templates/.github/ISSUE_TEMPLATE/feature_request.yml

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+name: Bug Report
+description: Report a bug in OVHcloud Control Panel application
+title: '<title>'
+labels: ['bug']
+body:
+  - type: checkboxes
+    attributes:
+      label: Have you already contact our support center?
+      description: Before to open an issue on our GitHub repository, did you already contacted our support center? (See https://help.ovhcloud.com)
```
Comment (marie-j) [inline .github/ISSUE_TEMPLATE/bug_report.yml:9]:
```suggestion
      description: Before opening an issue on our GitHub repository, have you already contacted our support center? (See https://help.ovhcloud.com)
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      options:
+        - label: Yes, I have contact the support center.
```
Comment (marie-j) [inline .github/ISSUE_TEMPLATE/bug_report.yml:11]:
```suggestion
        - label: Yes, I have contacted the support center.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+          required: true
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
```
Comment (marie-j) [inline .github/ISSUE_TEMPLATE/bug_report.yml:18]:
```suggestion
        - label: I have checked the existing issues
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+          required: true
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
```
Comment (marie-j) [inline .github/ISSUE_TEMPLATE/bug_report.yml:23]:
```suggestion
      description: A clear and concise description of the bug.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. In this environment...
+        2. With this config...
+        3. Run '...'
+        4. See error...
```
Comment (marie-j) [inline .github/ISSUE_TEMPLATE/bug_report.yml:34]: Quite like that "As a user I should be able to" but applied to Bug :)

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+      label: What device are you seeing the problem on?
+      options:
+        - Desktop
+        - Tablet
+        - Smartphone
+  - type: textarea
+    attributes:
+      label: Additional information to add?
+      description: |
+        Add any other context about the problem here. (Screenshots, links, etc…)
```
Comment (marie-j) [inline .github/ISSUE_TEMPLATE/bug_report.yml:66]:
```suggestion
        Add any other relevant information about the problem here. (Screenshots, links, etc…)
```
Maybe we should also specify that information should be anonymized ? No data ?

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+      required: true
+  - type: textarea
+    attributes:
+      label: Describe the solution
+      description: A clear and concise description of what you want to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Describe alternatives you've considered
```
[the remark this answers is not in the document]
Comment (JeremyDec) [reply .github/ISSUE_TEMPLATE/feature_request.yml:27]: Yes I've tried to mix the markdown items, with new ones introduced with this template :)

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 8]
```diff
@@ -0,0 +1,36 @@
+name: Feature request
+description: Suggest an idea for the OVHcloud Control Panel application
+title: '<title>'
+labels: ['enhancement']
+body:
+  - type: checkboxes
+    attributes:
+      label: Have you already contact our support center?
```
[the remark this answers is not in the document]
Comment (JeremyDec) [reply .github/ISSUE_TEMPLATE/feature_request.yml:8]: Good point, it is named as "Help centre" if we check https://help.ovhcloud.com/en/

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 9]
```diff
@@ -0,0 +1,70 @@
+name: Bug Report
+description: Report a bug in OVHcloud Control Panel application
+title: '<title>'
+labels: ['bug']
+body:
+  - type: checkboxes
+    attributes:
+      label: Have you already contact our help centre?
```
Comment (marie-j) [inline .github/ISSUE_TEMPLATE/bug_report.yml:8]:
```suggestion
      label: Have you already contacted our help centre?
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+      description: Before opening an issue on our GitHub repository, have you already contacted our help centre? (See https://help.ovhcloud.com)
+      options:
+        - label: Yes, I have contacted the help centre.
+          required: false
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please check an issue already exists for the bug you encountered.
```
Comment (marie-j) [inline .github/ISSUE_TEMPLATE/bug_report.yml:16]:
```suggestion
      description: Please check if an issue already exists for the bug you encountered.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+      multiple: true
+      options:
+        - Desktop
+        - Tablet
+        - Smartphone
+  - type: textarea
+    attributes:
+      label: Additional information to add?
+      description: |
+        Add any other relevant information about the problem here. (Screenshots, links, etc…). Be careful to not display any sensistive or personal information
```
Comment (marie-j) [inline .github/ISSUE_TEMPLATE/bug_report.yml:67]:
```suggestion
        Add any other relevant information about the problem here. (Screenshots, links, etc…). Be careful to not display any sensitive or personal information
```

**Review by antleblanc — APPROVED** [review]

**Pull request merged**